source/kubernetes/direct-csi.rst

```diff
@@ -0,0 +1,135 @@
+==============================================
+MinIO Direct Container Storage Interface (CSI)
+==============================================
+
+.. default-domain:: minio
+
+.. contents:: On This Page
+   :local:
+   :depth: 2
+
+Overview
+--------
+
+The MinIO Direct Container Storage Interface (CSI) driver for Kubernetes
+supports creating Persistant Volumes (``PV``) from disks attached to a 
+:kube-docs:`Node <concepts/architecture/nodes/>`. The
+:doc:`MinIO Kubernetes Operator </kubernetes/operator-reference>` and
+:doc:`MinIO Kubernetes Plugin </kubernetes/kubectl-plugin-reference>`
+support using MinIO Direct CSI volumes when creating or expanding
+MinIO deployments. 
+
+Complete documentation on the :kube-docs:`Container Storage Interface
+<concepts/storage/volumes/#csi>` is out of scope for this documentation. 
```

Comment:
Rephrase as "Complete Documentation on the.. CSI.. can be found here"? Saying it is out of scope seems redundant with the text below.

Reply:
I've got no objections to that construction - I'll use that as the standard approach.